### ISSUE TITLE
Examples of stacked vertical items

### ISSUE THREAD

**Issue by rlightner** [bkzl/vue-fraction-grid#5]
I'm trying to do a 1/3 , 2/3 grid with the 2/3 containing a 3/4, 1/4 stacked vertical grid. The items are not taking up 100% of the width. Can you add some examples of stacked vertical?

**Comment by bkzl** [comment]
Sorry, I don't exactly get what do you want to achieve. Could you attach a simple drawing? Do you mean something like this?

![screen shot 2017-03-12 at 14 46 17](https://cloud.githubusercontent.com/assets/127219/23832368/d34896a8-0732-11e7-8bb6-e87e5c4aa24a.png)

```html
<container>
  <grid>
    <grid-item size="1/3">
      1
    </grid-item>
    <grid-item size="2/3">
      2
      <grid>
        <grid-item size="3/4">3</grid-item>
        <grid-item size="1/4">4</grid-item>
      </grid>
    </grid-item>
  </grid>
</container>
```

Or that?

![screen shot 2017-03-12 at 14 54 38](https://cloud.githubusercontent.com/assets/127219/23832421/e9e65f52-0733-11e7-8e02-4a7b6dfce4f0.png)

The second one is currently not possible without using custom css (size prop only modifies the width).

**Comment by rlightner** [comment]
Yes, second option is what I was looking for. Flex supports this, does it not?

**Comment by bkzl** [comment]
Yes it does, but plugin's `size` only sets the width of a `<grid-item>`. I'll think about the additional API for height. So right now you have to write own CSS for that case or use `grow` as a workaround:

![screen shot 2017-03-13 at 11 42 38](https://cloud.githubusercontent.com/assets/127219/23851355/84c7e73e-07e3-11e7-9ae1-1aa4dac29537.png)

```html
<container>
  <grid>
    <grid-item size="1/3">1</grid-item>
    <grid-item size="2/3">
      <grid :style="{height: '200px'}" direction="stack">
        <grid-item grow="3">2</grid-item>
        <grid-item grow="1">3</grid-item>
      </grid>
    </grid-item>
  </grid>
</container>
```

**Comment by rlightner** [comment]
Gotcha, thanks again! This works really well with Keen-UI.
